File: Testing/Stdlib Test Code/test_multi_return_out.js
function minmax(a, b) {
    if ((a < b)) {
        return [a, b];
    }
    return [b, a];
}

function main() {
    let [lo, hi] = minmax(10, 3);
    console.log(lo);
    console.log(hi);
    return 0;
}


main();

File: Testing/Stdlib Test Code/test_multi_return_out.py
from dataclasses import dataclass, field


def minmax(a, b):
    if (a < b):
        return (a, b)
    return (b, a)

def main():
    lo, hi = minmax(10, 3)
    print(lo)
    print(hi)
    return 0


if __name__ == "__main__":
    main()
